src/AppBundle/Utils/Markdown.php

```diff
@@ -34,7 +34,11 @@ class Markdown
     public function __construct()
     {
         $this->parser = new \Parsedown();
-        $this->purifier = new \HTMLPurifier();
+
+        $purifierConfig = \HTMLPurifier_Config::create(array(
+            'Cache.DefinitionImpl' => null, // Disable caching
```

Comment:
@bocharsky-bw, it will require more code for creating those directories.
See https://github.com/Exercise/HTMLPurifierBundle/blob/master/CacheWarmer/SerializerCacheWarmer.php